src/main/java/io/sealights/plugins/sealightsjenkins/buildsteps/cli/LogConfiguration.java

```diff
@@ -41,7 +48,9 @@
                 properties.add(formatKeyValue(SL_LOG_FILE_NAME, logFilename));
             }
             if (!StringUtils.isNullOrEmpty(logFolder)) {
-                properties.add(formatKeyValue(SL_LOG_FOLDER, logFolder));
+                String resolvedLogFolder = resolveLogFolder(logFolder);
+                logger.info("Log folder set to " + resolvedLogFolder);
```

Comment:
Please quote the value of `resolvedLogFolder`

Reply:
Done